pkg/validation/validation.go

```diff
@@ -28,9 +28,17 @@ var CustomResourceDefinitionValidator = internal.CRDValidator
 var BundleValidator = internal.BundleValidator
 
 // OperatorHubValidator implements Validator to validate bundle objects
-// for OperatorHub.io requirements.
+// for OperatorHub.io requirements. This validator is deprecated.
 var OperatorHubValidator = internal.OperatorHubValidator
 
+// OperatorHubValidatorV2 implements Validator to validate bundle objects
+// for OperatorHub.io requirements.
+var OperatorHubValidatorV2 = internal.OperatorHubValidatorV2
```

Comment:
Nit: seems more consistent to put `Validator` as the suffix?
```suggestion
var OperatorHubV2Validator = internal.OperatorHubValidatorV2
```